config-model/src/main/java/com/yahoo/vespa/model/content/ContentNode.java

```diff
@@ -25,6 +25,10 @@ public abstract class ContentNode extends AbstractService
     private final boolean dispatch_on_decode;
     private final int mbus_threads;
     private final int mbus_network_threads;
+    private final int mbus_cpp_rpc_targets;
+    private final int mbus_events_before_wakeup;
```

Comment:
I think it would make sense to rename `mbus_cpp_rpc_targets` to just `mbus_rpc_targets` to be in line with `mbus_events_before_wakeup`. C++ is already implied since it's the content node service

Reply:
Fixed